### ISSUE TITLE
pygments style shows up on config styles but cannot be used

### ISSUE THREAD

**Issue by ypekka** [xonsh/xonsh#1097]
Installing pygments style from pypi doesn't  work fully.

Installing package "pygments-style-solarized" adds "solarizedlight" and "solarizeddark"  styles, they show up with "xonfig styles", but following exception is raised when used in config.json.

% /opt/xonsh/bin/xonsh
/opt/xonsh/lib/python3.5/site-packages/xonsh/pyghooks.py:306: RuntimeWarning: Could not find style 'solarizeddark', using default
  RuntimeWarning)

**Comment by scopatz** [comment]
Thanks for reporting this @ypekka! There are sort of two things going on here:

1. We would need to add these styles to [`xonsh.pyghooks`](https://github.com/scopatz/xonsh/blob/master/xonsh/pyghooks.py) and probably also `ansi_colors`
2. Maybe we should allow the pygments color style to work and use the default style if the style is allowed by pygments.

Pull requests for both of these are most welcome!

**Comment by stockninja** [comment]
While trying to replicate this, I may have unearthed another issue. 

When I run `$XONSH_COLOR_STYLE = 'foo'`, my whole terminal window crashes. Happens in both Terminal.app and iTerm2.app on OSX.

Tried setting `XONSH_DEBUG = 1` to see if I could catch an exception, but that didn't help.

**Comment by scopatz** [comment]
@stockninja - ahh yeah, that is a separate issue.  We should probably issue a warning if the color style doesn't exist.  I actually thought we did this already.... Try running xonsh from within another xonsh or bash or etc instance to see the full traceback

**Comment by airdrik** [comment]
in my personal opinion, I would prefer to leave the ANSI colors separate from the syntax highlighting.  

I found it a little odd that the colors in my prompt (defined using ANSI colors) changed each time I switched style.  I am also running into a problem, which I'm inclined to believe to be related, where the nano text editor is basically unusable as most things are showing up as dark-gray on dark-gray (doesn't matter which style I use.  Running a new bash shell and running nano from there produces the expected coloration).

So my vote would be to just use pygments for the syntax highlighting bits, and leave the ANSI colors alone.
